vtex-apps/order-items
# CHK-609: Contextual pipeline for adding/updating items

#### What problem is this solving?

Adding the new parameters that allow the contextual pipeline execution

#### How to test it?

[Here](https://contextualpipeline--checkoutio.myvtex.com/_v/private/vtex.order-items@0.11.0/graphiql/v1?query=mutation%20%7B%0A%20%20updateItems(orderItems%3A%20%7B%0A%20%20%20%20index%3A%200%2C%0A%20%20%20%20quantity%3A%202%0A%20%20%7D%2C%0A%20%20allowedOutdatedData%3A%20%5B%22paymentData%22%5D)%7B%0A%20%20%20%20value%0A%20%20%20%20paymentData%7B%0A%20%20%20%20%20%20installmentOptions%7B%0A%20%20%20%20%20%20%20%20value%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D)
Change the quantity in the mutation and run it. The `value` fields in `installmentOptions` should not be equal to the `value`field

#### Screenshots or example usage:

#### Describe alternatives you've considered, if any.

#### Related to / Depends on

[checkout-resources](https://github.com/vtex-apps/checkout-resources/pull/71)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/Zvx9ttT97cevkbyVik/giphy.gif)
